### PR TITLE
DrawSurfaceToTikz Documentation

### PR DESCRIPTION
**Pull request by MeikeWeiss**
Issue #59, Issue #58 and some ToDos

### CODE REVIEW

File: gap/PolygonalComplexes/drawing.gd [hunk 1]
```diff
@@ -344,20 +357,81 @@ DrawSurfaceToTikz( tetra, "Tetrahedron_faceLabels", pr);;
...
 #! polygons their edge lengths and inner angles have to be defined (exception:
 #! for triangles it is sufficient to define the edge lengths, as long as the
-#! angles are not set). An example can be seen at the start of this section
-#! (<Ref Sect="Section_Embeddings_DrawTikz"/>).
+#! angles are not set).
 #!
 #! If the given information is inconsistent an error is thrown.
-#TODO example?
+#!
+#! We will exemplify the first two  parameters on the example of a tetrahedron. Here the tetrahedron has holes in the
```
Comment (ReymondAkpanya) [inline gap/PolygonalComplexes/drawing.gd:373]: there is a unnecessary blank between "two" and "parameters"

File: gap/PolygonalComplexes/drawing.gd [hunk 2]
```diff
@@ -371,18 +445,68 @@ DrawSurfaceToTikz( tetra, "Tetrahedron_faceLabels", pr);;
...
+#! <Alt Only="TikZ">
+#!    \input{_TIKZ_Butterfly_startingFaces.tex}
+#! </Alt>
+#! </Item>
+#! <Item><E>edgeDrawOrder</E>: After drawing this list contains a list of edges for each 
+#! strongly connected component. For each component, this list explains the order in which 
+#! these edges were completed (an edge is completed if both adjacent faces are drawn).
+#! This order can be changed with this parameter.
+#!
+#! For example, consider this simplicial surface:.
```
Comment (ReymondAkpanya) [inline gap/PolygonalComplexes/drawing.gd:490]: unnecessary dot after "surface:"

**Review by ReymondAkpanya — CHANGES_REQUESTED** [review]

**Review by ReymondAkpanya — DISMISSED** [review]

File: gap/PolygonalComplexes/drawing.gd [hunk 3]
```diff
@@ -145,13 +149,14 @@ DrawSurfaceToTikz(cube, "Cube_example");;
...
-#!     colour of the vertex <A>v</A>. The colours are strings that are 
+#!     colour of the vertex <A>v</A>. If vertex <A>v</A> is not in the surface, 
+#!     the colour is skipped. The colours are strings that are 
 #!     recognizable by &LaTeX;.
 #!
 #!     It is possible to leave some vertex colours unbound - those will be coloured with the
 #!     default vertex colour (orange).
 #! @BeginLog
-pr := rec( vertexColours := ["red", "blue", "green",, "black!20!yellow"] );;
+pr := rec( vertexColours := ["red", "blue", "green",, "pink", "black!20!yellow"] );;
```
Comment (TomGoertzen) [inline gap/PolygonalComplexes/drawing.gd:159]: You could add a line-break here

File: gap/PolygonalComplexes/drawing.gd [hunk 4]
```diff
@@ -171,13 +176,14 @@ DrawSurfaceToTikz(cube, "Cube_vertexColouredGlobal", pr);;
...
-#!     is the colour of the edge <A>e</A>. The colours are strings that are
+#!     is the colour of the edge <A>e</A>. If edge <A>e</A> is not in the surface,
+#!     the colour is skipped. The colours are strings that are
 #!     recognizable by &LaTeX;.
 #!
 #!     Is is possible to leave some edge colours unbound - those will be coloured with the
 #!     default edge colour (gray).
 #! @BeginLog
-pr := rec( edgeColours := [,,,,"red", "blue", "purple", "green!80!black"] );;
+pr := rec( edgeColours := [,,,,"red", "purple", "blue", "green!80!black"] );;
```
Comment (TomGoertzen) [inline gap/PolygonalComplexes/drawing.gd:186]: Also a line-break here

File: gap/PolygonalComplexes/drawing.gd [hunk 5]
```diff
@@ -371,18 +445,68 @@ DrawSurfaceToTikz( tetra, "Tetrahedron_faceLabels", pr);;
...
+#!   completed if both adjacent faces are drawn). This means that this parameter 
+#!   controls which edges are drawn once and which edges need to be drawn twice because 
+#!   they cannot be completed. This can be customized and
 #!   <K>DrawSurfaceToTikz</K> will try to follow it as closely as possible
 #!   while avoiding intersections (if <E>avoidIntersections</E> is set).
 #! 
-#! To see this in action, we consider TODO
+#! We will exemplify these parameters on different examples.
+#! <List>
+#! <Item><E>startingFaces</E>: After drawing this list contains the first faces that were drawn from
```
Comment (TomGoertzen) [inline gap/PolygonalComplexes/drawing.gd:464]: Maybe add a comma after "After drawing"

**Review by TomGoertzen — COMMENTED** [review]
Great pictures. Please take a look at my comments

**Review by TomGoertzen — APPROVED** [review]

**Review by ReymondAkpanya — APPROVED** [review]

**Pull request merged**